### PR TITLE
Prefer the Homebrew official formula

### LINKED ISSUE #56: Support Homebrew Bottles
Support [Bottles](https://github.com/Homebrew/brew/blob/master/docs/Bottles.md) and expand `depends_on` of Homebrew Formula.

### LINKED ISSUE #88: Integrate into main homebrew repository
This is a great little tool, thanks for the effort.

I noticed the homebrew installation is via a tap, would you consider adding the formula to the main homebrew repository?

I assume they generally want the formulas added by the maintainer where possible, but otherwise, I can try and send a PR for that.

Or have you already considered that and decided it's not what you want?

### PR DESCRIPTION
**Pull request by umireon**
LicensePlist is now included on Homebrew and can be installed via `brew install licenseplist`.

I suggest that we prefer the Homebrew official formula.

Official Homebrew formula installs the pre-built binary so Homebrew users don't need to compile from the source.

Closes #56 
Closes #88

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +34,7 @@ pod 'LicensePlist'
...
 ```sh
-brew install mono0926/license-plist/license-plist
-```
-
-Or
-
-```sh
-brew tap mono0926/license-plist
-brew install license-plist
+brew install licenseplist
```
Comment (mono0926) [inline README.md:37]: I'd like to keep to `license-plist`, not `licenseplist` 

Why did you change this?

**Review by mono0926 — COMMENTED** [review]
@umireon 

As  I wrote at https://github.com/mono0926/LicensePlist/issues/88#issuecomment-398938973,  I'm not familiar with Homebrew distribution.

Should I maintain https://github.com/Homebrew/homebrew-core/blob/master/Formula/licenseplist.rb in the future?
If so, I think the release script( https://github.com/mono0926/LicensePlist/blob/main/release.sh ) should be changed to do a pull request to update that as well.

**Comment by umireon** [comment]
You don't need to maintain https://github.com/Homebrew/homebrew-core/blob/master/Formula/licenseplist.rb.
Homebrew users will update this code.

File: README.md (hunk 1 above)
Comment (umireon) [reply README.md:37]: I followed the Homebrew's convention and your original naming was not followed it.

If the name of the repository is `mono0926/LicensePlist`, the name of the formula should be `licenseplist`, and if that of the repository is `mono0926/license-plist`, the name should be `license-plist`.

I don't know if Homebrew maintainers would accept this exception.

Comment (mono0926) [reply README.md:37]: @umireon 

I see, then I thought it would be okay 👌

**Comment by umireon** [comment]
@mono0926 Feel free to ask me if somebody complains about the installation on Homebrew. I can fix it.
